src/lib/stitching/exchange/v2/stitching.ts

```diff
@@ -349,6 +356,87 @@ export const exchangeStitchingEnvironment = ({
           },
         },
       },
+      Mutation: {
+        createInquiryOffer: {
+          resolve: async (_source, args, context, info) => {
+            const {
+              conversationLoader,
+              conversationCreateConversationOrderLoader,
+            } = context
+            const {
+              input: { impulseConversationId },
+            } = args
+
+            try {
+              await conversationLoader(impulseConversationId)
```

Comment:
Am I missing something, or is there a return value of this missing? Or is it just doing an exist-y check?

Reply:
just an exist-y check - though we could make more explicit what is going on.